transport-native-epoll/src/main/c/io_netty_channel_unix_Errors.c

```diff
@@ -32,14 +34,10 @@ static char* exceptionMessage(char* msg, int error) {
         // and it's hard to keep track of when to send -error and when not
         // this will just take care when things are forgotten
         // what would generate a proper error
-        error = error * -1;
+        error = -error;
     }
     //strerror is returning a constant, so no need to free anything coming from strerror
-    char* err = strerror(error);
-    char* result = malloc(strlen(msg) + strlen(err) + 1);
-    strcpy(result, msg);
-    strcat(result, err);
-    return result;
+    return netty_unix_util_prepend(msg, strerror(error));
```

Comment:
@normanmaurer - Also note this change. Just found that is it doing the same thing as `netty_unix_util_prepend`